File: code-coaches/medium/duty-free/DutyFree.cs
using System;

public class DutyFree
{
    static bool IsOnBudget(double[] itemsPriceEur)
    {
        double usdEurRate = 1.1;
        double limitPriceUsd = 20;

        foreach (double itemPriceEur in itemsPriceEur)
        {
            double itemPriceUsd = itemPriceEur * usdEurRate;
            if (itemPriceUsd > limitPriceUsd)
            {
                return false;
            }
        }

        return true;
    }

    static void Main(string[] args)
    {
        // Read item prices from standard input
        string[] input = Console.ReadLine().Split();
        double[] itemsPriceEur = Array.ConvertAll(input, double.Parse);

        // Check if on budget and output result
        if (IsOnBudget(itemsPriceEur))
            Console.WriteLine("On to the terminal");
        else
            Console.WriteLine("Back to the store");
    }
}

File: code-coaches/medium/duty-free/duty_free.py
def is_on_budget(items_price_eur: list[float]) -> bool:
    usd_eur_rate = 1.1
    limit_price_usd = 20

    for item_price_eur in items_price_eur:
        item_price_usd = item_price_eur * usd_eur_rate
        if item_price_usd > limit_price_usd:
            return False

    return True


def main():
    items_price_eur = list(map(float, input().split()))

    if is_on_budget(items_price_eur):
        print("On to the terminal")
    else:
        print("Back to the store")


if __name__ == "__main__":
    main()
